onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -513,6 +514,28 @@ def testProfilerWithSessionOptions(self):
                     self.assertTrue(tag in lines[i])
             self.assertTrue(']' in lines[8])
 
+    def testProfilerGetStartTimeNs(self):
+        # Get current nanoseconds
+        start = time.monotonic_ns()
+        x = np.array([[1.0, 2.0], [3.0, 4.0], [5.0, 6.0]], dtype=np.float32)
+        
+        # Get 1st profiling's start time
+        so = onnxrt.SessionOptions()
+        so.enable_profiling = True
+        sess = onnxrt.InferenceSession(get_name("mul_1.onnx"), sess_options=so)
+        sess.run([], {'X': x})
+        start_time_1 = sess.get_profiling_start_time_ns()
+        
+        # Get 2nd profiling's start time
+        so = onnxrt.SessionOptions()
+        so.enable_profiling = True
+        sess = onnxrt.InferenceSession(get_name("mul_1.onnx"), sess_options=so)
+        sess.run([], {'X': x})
+        start_time_2 = sess.get_profiling_start_time_ns()
+
+        # Chronological profiling's start time
+        self.assertTrue(start <= start_time_1 <= start_time_2)
```

Comment:
Can these start times actually equal to each other?

Reply:
Practically these times should not be equal, but logically if the processing power is strong enough, it's "possible" that these times are equal. Since the validation here is to simply assert these times are chronological, I chose to use equal here. In that case, this loose condition won't fail in any chance under the correct functionality.